commit 11c5e7f766ff4ddcac6d51ef65fabd362b092ac2 (refs/changes/16/116/5)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2016-07-22T22:43:05+00:00 (3 years, 3 months ago)
    
    OS-5535 refresh platform login banner with Triton branding
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/overlay/generic/etc/issue b/overlay/generic/etc/issue
index d9851bc6..d9957b7a 100644
--- a/overlay/generic/etc/issue
+++ b/overlay/generic/etc/issue
@@ -1,10 +1,14 @@
 
-             _____
-          ____   ____
-         _____   _____        .                   .
-         __         __        | .-. .  . .-. :--. |-
-         _____   _____        ;|   ||  |(.-' |  | |
-          ____   ____     `--'  `-' `;-| `-' '  ' `-'
-             _____                  /  ; Joyent Live Image v0.147+
-                                    `-'   build: 00000000T000000Z
+
+         [37m[1m*[31m--[0m[31m+--[37m[1m*[0m[33m--[37m[1m*[0m
+         [31m|[1m\[0m [31m|[0m[31m\ |[33m\ [1m|\[0m      J O Y E N T
+         [31m| [1m\[0m[31m|[0m [31m\| [33m\[1m| \[0m     [37;47m#####[0m  [37;47m####[0m   [37;47m#[0m  [37;47m#####[0m  [37;47m###[0m   [37;47m#[0m   [37;47m#[0m TM
+         [31m+--[1m[37m*[31m--+[0m[33m--[1m[37m*[33m--[37m*[0m      [37;47m#[0m    [37;47m#[0m   [37;47m#[0m  [37;47m#[0m    [37;47m#[0m   [37;47m#[0m   [37;47m#[0m  [37;47m##[0m  [37;47m#[0m
+         [37m|[31m\ |\ |\ [33m|[1m\ |[0m      [37;47m#[0m    [37;47m####[0m   [37;47m#[0m    [37;47m#[0m   [37;47m#[0m   [37;47m#[0m  [37;47m#[0m [37;47m#[0m [37;47m#[0m
+         [37m| [31m\| \| \[33m| [1m\|[0m      [37;47m#[0m    [37;47m#[0m  [37;47m#[0m   [37;47m#[0m    [37;47m#[0m   [37;47m#[0m   [37;47m#[0m  [37;47m#[0m  [37;47m##[0m
+         [37m[1m*[0m[37m--+[31m--+[33m--+[1m--+[0m      [37;47m#[0m    [37;47m#[0m   [37;47m#[0m  [37;47m#[0m    [37;47m#[0m    [37;47m###[0m   [37;47m#[0m   [37;47m#[0m
+          [37m[1m\ |[0m[34m\ |\ |[1m\ |[0m
+           [37m[1m\| [0m[34m\| \| [1m\|[0m     [1;37mSmartOS[0m
+            [37m[1m*--+[0m[34m--[37m[1m*[34m--[37m*[0m      build: 00000000T000000Z
+
 
diff --git a/overlay/generic/etc/motd b/overlay/generic/etc/motd
index 1ef66a68..23644ffa 100644
--- a/overlay/generic/etc/motd
+++ b/overlay/generic/etc/motd
@@ -1 +1 @@
-- SmartOS Live Image v0.147+ build: 00000000T000000Z
+- SmartOS (build: 00000000T000000Z)
